server/src/seeder/seeder.module.ts

```diff
@@ -32,9 +45,16 @@ export class SeederModule implements OnApplicationBootstrap {
     Logger.log('onApplicationBootstrap', SeederModule.name);
 
     // remove all data
-    this.seederService.truncateFromAllTables();
+    //this.seederService.truncateFromAllTables();
 
     // seed the users table
-    await this.seederService.seedUsersAsync();
+    this.seederService
+      .seedUsersAsync()
+      .then(() => this.seederService.seedAssetsAsync())
```

Comment:
in terms of creating the relationships, i think if we make these async seeding service functions return the data that we get back from the `.create` functions, we should be able to access that data in the next `.then`'s callback
aka:
we return the new user in `seedUsersAsync`
line above changes to `.then((newUser) => this.seederService.seedAssetsAsync(newUser))`
and then creating the relationship with that user is easy
hope that works!

Reply:
updated to pass results from previous promise.